### DISCUSSION TITLE
How to add custom Font ?

### DISCUSSION THREAD

**Discussion by ds2k5** [templaza/astroid-framework#902] [General]
Hi,
tryed to add Custom font to the template.
But did not work. Can not select the Font "gotyk-webfont"

System -> Site Template Stile -> astroid_template_one - Standard Astroid -> Template Options -> Typography

![grafik](https://github.com/user-attachments/assets/f1a673f4-bc30-4f6a-b025-ad0ab9058445)

/var/www/html/templates/astroid_template_one/custom.css

```
@font-face {
    font-family: 'gotyk-webfont';
    src: url('../fonts/gotyk-webfont.woff2') format('woff2'),
         url('../.gotyk-webfont.woff') format('woff');
    font-weight: normal;
    font-style: normal;
}

```

![grafik](https://github.com/user-attachments/assets/f9d20f1e-51a5-453b-9c5f-d39e1693f7ee)

Font: https://www.dafont.com/gotyk-poszarpany.font

![grafik](https://github.com/user-attachments/assets/ee0f7ac8-2ae4-4cd8-af1f-39afafdbb8f5)

**Comment by Banuk1616** [comment]
Hello
It is quite simple. Download the font. Create a “fonts” folder in the root/media/templates/site/your-template folder and save your font in this folder. The font is then installed locally and you can select the font in Astroid under Typography options.
Best Regards

**Comment by ds2k5** [comment]
@Banuk1616 

Thanks a lot ! works!

**Comment by Banuk1616** [comment]
I am pleased that I was able to help you. Please set the thread to resolved.
Best Regards
<img width="821" alt="resolved" src="https://github.com/user-attachments/assets/298e66f0-c2e2-4ddb-b233-4a992aaccbdb">

**Comment by ds2k5** [comment]
thanks @Banuk1616
